service/src/test/java/bio/terra/workspace/service/iam/SamServiceTest.java

```diff
@@ -225,11 +221,11 @@ void nonOwnerCannotAddReader() {
   void permissionsApiFailsInRawlsWorkspace() throws Exception {
     UUID workspaceUuid = UUID.randomUUID();
     // RAWLS_WORKSPACEs do not own their own Sam resources, so we need to manage them separately.
-    samService.createWorkspaceWithDefaults(defaultUserRequest(), workspaceUuid);
+    samService.createWorkspaceWithDefaults(defaultUserRequest(), workspaceUuid, new ArrayList<>());
 
     Workspace rawlsWorkspace =
         WorkspaceFixtures.buildWorkspace(workspaceUuid, WorkspaceStage.RAWLS_WORKSPACE);
-    workspaceService.createWorkspace(rawlsWorkspace, null, null, defaultUserRequest());
+    workspaceService.createWorkspace(rawlsWorkspace, null, null, null, defaultUserRequest());
```

Comment:
It would have been easier to make a `createWorkspaceWithPolicy` method and avoid adding all of the nulls to all of the tests.